x-pack/plugins/apm/public/components/app/ServiceMap/icons.ts

```diff
@@ -89,7 +89,7 @@ const agentIcons: { [key: string]: string } = {
 function getAgentIcon(agentName?: string) {
   // RUM can have multiple names. Normalize it
   const normalizedAgentName = isRumAgentName(agentName) ? 'js-base' : agentName;
-  return normalizedAgentName && agentIcons[normalizedAgentName];
+  return normalizedAgentName && agentIcons[normalizedAgentName.toLowerCase()];
 }
```

Comment:
Would it make sense to extract and move to `common/agent_name.ts`?

```ts
function getNormalizedAgentName(agentName?: string) {
  // RUM can have multiple names. Normalize it
  const normalizedAgentName = isRumAgentName(agentName) ? 'js-base' : agentName;
  return normalizedAgentName?.toLowerCase();
}
```

Reply:
Updated in 1f1fa45.